pkg/rancher-desktop/pages/marketplace/details.vue

```diff
@@ -0,0 +1,352 @@
+<template>
+  <div class="extension">
+    <nuxt-child />
+    <div class="extension-content">
+      <div>
+        <Banner v-if="error" color="error" class="banner">
+          {{ error }}
+        </Banner>
+        <Banner v-if="response" color="info" class="banner">
+          {{ response }}
+        </Banner>
+        <div v-html="htmlContent" />
+
+        <img :src="getPreviewImage" alt="" />
+      </div>
+      <div class="extension-content-sidebar">
+        <button
+          v-if="!isInstalled"
+          class="btn btn-xs role-primary"
+          @click="appInstallation()"
+        >
+          <slot v-if="loading" name="loading" :is-loading="loading">
```

Comment:
I don't think that it makes sense to expose a slot here. Do we anticipate that the parent will need to customize the loading indicator?